### DISCUSSION TITLE
commodifying outfits for transport

### DISCUSSION THREAD

**Discussion by flowers9** [endless-sky/endless-sky#9149] [Ideas]
I've been thinking about a way to transport outftits that are put into cargo at an Outfitter that treat them more like commodities, in that they become a bunch of 1 ton bits that can be easily moved around in ship cargos.  Outfits would be commodified during takeoff and reassembled into outfits when landing.  Partial outfits (if you should lose a ship, or if not all ships holding it are landed) would be left on the ship (and could be dumped from there).

It makes sense to me that outfits wouldn't be transported fully assembled, but rather in a more easily transported method, and this would help deal with the issue of outfits not fitting neatly into cargo space when you've got a fleet.

**Comment by Hecter94** [comment]
This is assuming all outfits can be easily dissembled and reassembled, which may be true for some basic human outfits, but a human crew being able to easily disassemble and reassemble something like a Jump Drive or Antimatter Reactor would cause some lore conflict I think.

**Reply by flowers9** [reply]
I dunno - everyone has the same basic needs, as far as transporting outfits goes.  If anything, I'd expect automation to make things even more modular (which might help explain the easy plug and play nature of looted outfits).

**Reply by mOctave** [reply]
Well, the Quarg probably make it _intentionally_ difficult to not take apart their outfits.
